api/src/feeds/impl/models/basic_feed_impl.py

```diff
@@ -61,4 +61,3 @@ class Config:
         Enabling `from_orm` method to create a model instance from a SQLAlchemy row object."""
 
         from_attributes = True
-        orm_mode = True
```

Comment:
This is unrelated, but it is causing a warning at the start time that can become an error. We have already removed all other places where we used this attribute.